crates/ruff_linter/src/rules/flake8_pyi/rules/string_or_bytes_too_long.rs

```diff
@@ -92,8 +93,36 @@ fn count_f_string_chars(f_string: &ast::ExprFString) -> usize {
                 .elements
                 .iter()
                 .map(|element| match element {
-                    ast::FStringElement::Literal(string) => string.chars().count(),
-                    ast::FStringElement::Expression(expr) => expr.range().len().to_usize(),
+                    ast::FTStringElement::Literal(string) => string.chars().count(),
+                    ast::FTStringElement::Expression(expr) => expr.range().len().to_usize(),
+                })
+                .sum(),
+        })
+        .sum()
+}
+
+/// Count the number of visible characters in a t-string. This accounts for
+/// implicitly concatenated t-strings as well.
+fn count_t_string_chars(t_string: &ast::ExprTString) -> usize {
```

Comment:
Let's maybe add a TODO and remove t-string hadnling for now. We need to be careful here if the templates inferred type is different based on the interpolations.